com.unity.cinemachine/Documentation~/CinemachineFreeLookModifier.md

```diff
@@ -19,3 +19,10 @@ As an example, if the CinemachineCamera has a lens FOV of 60, and you add a lens
 
 > [!NOTE] 
 > [Save During Play](CinemachineSavingDuringPlay.md) tracks changes to properties of the existing items in the Modifiers list. However, it doesn't track any addition or removal of modifiers while in Play mode.
+
+## Properties:
+
+| **Property:** | **Function:** |
+|:---|:---|
+| __Easing__ | The amount of easing to apply towards the center value.  Zero easing will blend linearly through the center value, while an easing of 1 will smooth the result as it passes over the center value. |
+| __Modifiers__ | The list of modifiers which will modify camera settings as a function of the FreeLook's Vertical axis value. |
```

Comment:
Minor things. Ideally, we should also update the screenshot above to reflect the addition, but on the other hand, I didn't review this page yet in the overall refactor process, so that's up to you, I'll do it at some point.
```suggestion
| Property | Function |
| :--- | :--- |
| **Easing** | The amount of easing to apply towards the center value. Zero easing blends linearly through the center value, while an easing of 1 smooths the result as it passes over the center value. |
| **Modifiers** | The list of modifiers which modify camera settings as a function of the FreeLook's Vertical axis value. |

```